docs/book/getting-started/zenml-pro/core-concepts.md

```diff
@@ -0,0 +1,22 @@
+# ZenML Pro Core Concepts
+
+In ZenML Pro, there is a slightly different entity hierarchy as compared to the open-source ZenML
+framework. This document walks you through the key differences and new concepts that are only available for Pro users.
+
+![Image showing the entity hierarchy in ZenML Pro](../../.gitbook/assets/org_hierarchy_pro.png)
```

Comment:
i think the content in the image is too small to read comfortably because of how big the diagram is. maybe if we remove the Tenant 2 that 's just lying on the right side, we can shrink the size by a lot.